explorer/syntax/parser.ypp

```diff
@@ -1271,6 +1271,18 @@ interface_body:
       $$.push_back(
           arena->New<AssociatedConstantDeclaration>(context.source_loc(), $3));
     }
+| interface_body EXTENDS expression SEMICOLON
+    {
+      $$ = $1;
```

Comment:
Things like statement_list do a std::move. Although I see a lot of the interface code got added without a move, maybe it should get it? Maybe not in this change, since you're just being consistent.

Reply:
There are a bunch of these. I'll send you a separate PR handling all of them.